.github/workflows/build.yaml

```diff
@@ -1,6 +1,9 @@
-name: static-container-registry
+name: build
 
-on: push
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
most small repositories in scality where GH action is setup have a specific set of branches that matches our workflow.

you can refer to [this file](https://github.com/scality/xcore-external-deps/blob/ba5f27e1f0fa427207558351b2bfacd3250e3c00/.github/workflows/test.yaml#L3-L11)
you don't need the branches like `q/**` but we only want to run CI on branches like `development/**` etc and not other kind of branch like `my_branche_to_test_some_feature_not_matching_the_scality_branch_patern`
:slightly_smiling_face:

Reply:
there is no Bert-e or waterflow on this repo... so building on any branch should be OK for now, and we can change that once/if bert-e is setup?

Reply:
I don't think we'll setup bert-e on this one, don't need waterfall here I suppose.
let's keep it this way then.

have you ever realized that `s**t` is an anagram of `this`  :exploding_head: